Makefile

```diff
@@ -805,6 +829,51 @@ deb: install themispp_install themis_jni_install
 
 	@find $(BIN_PATH) -name \*.deb
 
+# TODO: Add Themis as dependency?
+# TODO: Put inside `deb` target?
+# TODO: Mention need of `lsb-release` in building instruction? Otherwise result file name is... eh... incomplete
+# Basically, this is just an unpacked version of a wheel.
+# Thus, extracting files from it to then put them in /usr/lib/python3.
+# Not just taking files from src/wrappers/themis/python because there are some files
+# that do not exist in the repo but are created by setup.py during wheel generation.
+pythemis_deb: DEB_ARCHITECTURE = all
+pythemis_deb: DESTDIR = $(BIN_PATH)/deb/pythemis_root
+pythemis_deb: pythemis_make_wheel
+	@# Remove old dir and unpack again to make sure there are only needed and fresh files.
+	@# Also, if not removing old files, will need some option for unzip to silently rewrite them instead of asking.
+	@rm -rf $(BIN_PATH)/deb/python3
+	@mkdir -p $(BIN_PATH)/deb/python3
+	@unzip src/wrappers/themis/python/dist/pythemis-$(VERSION_SHORT)-py2.py3-none-any.whl -d $(BIN_PATH)/deb/python3
+
+	@# Remove old one to avoid error "package already exist, refusing to proceed"
+	@rm -f $(BIN_PATH)/deb/python3-pythemis_$(NAME_SUFFIX)
+
+	@fpm --input-type dir \
+		 --output-type deb \
+		 --name python3-pythemis \
+		 --license $(LICENSE_NAME) \
+		 --url '$(COSSACKLABS_URL)' \
+		 --description '$(SHORT_DESCRIPTION)' \
+		 --maintainer $(MAINTAINER) \
+		 --package $(BIN_PATH)/deb/python3-pythemis_$(NAME_SUFFIX) \
+		 --architecture $(DEB_ARCHITECTURE) \
+		 --version $(VERSION)+$(OS_CODENAME) \
+		 --depends python3 --depends python3-six \
```

Comment:
>why we need python3-six?

Because it's mentioned in `src/wrappers/themis/python/setup.py`. Although, not sure we still need it as it's some [compatibility thing](https://pypi.org/project/six/) to make code with with Python 2 and 3.